pyne/ionization_calculator.py

```diff
@@ -0,0 +1,598 @@
+#Begin the method to calculate ionionization energies in eV for various atoms and electronic states
+
+from pyne import nucname
+import numpy as np
+
+def function(element,charge):
+ 
+#initialize tuples 
+  	
+
+#parse input
+index = nucname.znum(element)
```

Comment:
This line is not part of the "function" function because it hasn't been indented. It's just part of the file and would run as soon as the file is imported.